vllm/entrypoints/openai/api_server.py

```diff
@@ -226,6 +227,7 @@ async def create_chat_completion(request: ChatCompletionRequest,
             ignore_eos=request.ignore_eos,
             use_beam_search=request.use_beam_search,
             skip_special_tokens=request.skip_special_tokens,
+            spaces_between_special_tokens=spaces_between_special_tokens,
```

Comment:
```suggestion
            spaces_between_special_tokens=request.spaces_between_special_tokens,
```

Reply:
I would 100% love to take this suggestion but that's 81 characters long which violates the codestyle and causes the build to break.